prestodb/dbapi.py

```diff
@@ -74,23 +75,43 @@ class Connection(object):
     def __init__(self, *args, **kwargs):
         self._args = args
         self._kwargs = kwargs
+        self._transaction = None
+        self.isolation_level = kwargs.pop('isolation_level', None)
 
     def close(self):
         """Presto does not have anything to close"""
         # TODO cancel outstanding queries?
         pass
 
+    def start_transaction(self):
+        request = prestodb.client.PrestoRequest(*self._args, **self._kwargs)
+        request.transaction_id = 'NONE'
+        self._transaction = Transaction(request)
+        self._transaction.begin()
+        return self._transaction
+
     def commit(self):
-        """FIXME: Not supported yet"""
-        raise NotImplementedError
+        if self.isolation_level is None:
+            return
+        self._transaction.commit()
```

Comment:
Can we have sanity check on `_transaction`?

Reply:
Yes, having `Connection._transaction` set to `None` should never happen here. The additional check supports the path to starting the transaction in the constructor, or in a setter if `isolation_level` becomes a property.